pycm/pycm_overall_func.py

```diff
@@ -838,22 +792,22 @@ def overall_statistics(
     lambda_A = lambda_A_calc(classes, table, P, population)
     DF = DF_calc(classes)
     overall_jaccard_index = overall_jaccard_index_calc(list(
-        jaccard_list.values()))
+        kwargs["jaccard_list"].values()))
```

Comment:
Could we have `AttributeError` here in case that we did not give the function "jacard_list"?

Reply:
It's not necessary, because `overall_statistics` is a **system function** and it's only called once.